UPGRADE.md

```diff
@@ -1,4 +1,4 @@
-## Upgrade steps from v. 5.2.x to v. 5.3 for native/hybrid/react native apps
+## Upgrade steps from v. 5.3.x to v. 6.0 for native/hybrid/react native apps
```

Comment:
We should mention that there is a new cli for generating hybrid apps (forcehybrid) and another  one for generating react native apps (forcereact).
Should we mention the sfdx plugin as well?

Reply:
Will do.